instrumented-tests/Benchmark/Benchmark.xcodeproj/project.pbxproj

```diff
@@ -17,6 +17,8 @@
 		61999EC924071A760094866C /* LoggingBenchmarkTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 61999EC824071A760094866C /* LoggingBenchmarkTests.swift */; };
 		61AEBF062423D2AE00A8CB51 /* Datadog.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 61AEBF052423D2AE00A8CB51 /* Datadog.framework */; };
 		61AEBF072423D2AE00A8CB51 /* Datadog.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 61AEBF052423D2AE00A8CB51 /* Datadog.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
+		61C363CB2436674B00C4D4E6 /* _Datadog_Private.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 61C363C22436650C00C4D4E6 /* _Datadog_Private.framework */; };
+		61C363CC2436674B00C4D4E6 /* _Datadog_Private.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 61C363C22436650C00C4D4E6 /* _Datadog_Private.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```

Comment:
IMO that should not be; `Datadog_Private` shouldn't produce a standalone binary and it should only be an internal module

Reply:
The thing is that in `Benchmark` we do `@testable import Datadog` to measure isolated `I/O` operations. This `@testable import` was making the tests not compile as the private module was not seen. Maybe #68 will fix this, so I will look into this after we merge it into this PR. So far I found no better approach.

Reply:
if we could at least link it statically, we could save some time at launch time for our users

Reply:
Not possible 😞, static linking seems to be only possible for products, not targets in SPM:

https://developer.apple.com/documentation/swift_packages/product/library/librarytype